suite/runsuite_wrapper.pl

```diff
@@ -305,6 +305,7 @@
                 'finite_shared_bb_cache,cache_shared_bb_regen|common.nativeexec' => 1, # i#1807
                 'finite_shared_trace_cache,cache_shared_trace_regen|common.nativeexec' => 1, # i#1807
                 # We list this without any "options|" which will match all variations.
+                'api.rseq' => 1, # i#6612
```

Comment:
This is the Windows 64-bit list: the Linux one is below (~line 396).

Reply:
My bad, didn't see the outer condition. Fixed.